src/main/java/com/zilen/weather/controller/WeatherController.java

```diff
@@ -15,9 +18,14 @@ public WeatherController(WeatherService weatherService) {
         this.weatherService = weatherService;
     }
 
-    @GetMapping("/{cityName}")
+    @GetMapping("/city/{cityName}")
```

Comment:
it would be better to left the previous option, but {city} instead of {cityName}